src/components/Students/Students.js

```diff
@@ -0,0 +1,16 @@
+import { Student } from "../Student/Student";
+
+export function Students({ title, list = []}) {
+    return (
+        <>
```

Comment:
Please provide full tags
```suggestion
        <React.Fragment>
```